swift-sdk/IterableInboxViewController.swift

```diff
@@ -144,6 +144,7 @@ open class IterableInboxViewController: UITableViewController {
                 cell.iconImageView?.image = UIImage(data: data)
             } else {
                 cell.iconImageView?.backgroundColor = UIColor(hex: "EEEEEE") // loading image
+                cell.iconImageView?.image = nil
```

Comment:
Looks like an unrelated change.. Is it intentional?

Reply:
Yes. I think you already made this change in another branch.